lib/token.js

```diff
@@ -534,9 +534,13 @@ function parseFromUrl(sdk, url) {
   }
 
   var oauthParamsCookie = cookies.getCookie(config.REDIRECT_OAUTH_PARAMS_COOKIE_NAME);
-  if (!hash || !oauthParamsCookie) {
+  if (!hash) {
     return Q.reject(new AuthSdkError('Unable to parse a token from the url'));
   }
+
+  if (!oauthParamsCookie) {
+    return Q.reject(new AuthSdkError('Unable to parse OAuth redirect params cookie'));
```

Comment:
If this occurs, is it a parsing issue, or simply the fact that that cookie is not available?
Should there be a parsing issue, do we even reach this point in the code or is there an exception thrown by `js-cookie` that we need to handle?

Reply:
Good catch. We're checking below to see if the cookie can be parsed, while we want to check for its existence here.